e2e/fee_middleware_test.go

```diff
@@ -1,29 +1,37 @@
 package e2e
 
 import (
-	"os"
+	"context"
 	"testing"
 
+	"github.com/strangelove-ventures/ibctest/ibc"
 	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/v4/e2e/testsuite"
 )
 
 func TestFeeMiddlewareTestSuite(t *testing.T) {
 	suite.Run(t, new(FeeMiddlewareTestSuite))
 }
 
 type FeeMiddlewareTestSuite struct {
-	suite.Suite
+	testsuite.E2ETestSuite
 }
 
 func (s *FeeMiddlewareTestSuite) TestPlaceholder() {
-	tag, ok := os.LookupEnv("SIMD_TAG")
-	s.Require().True(ok)
-	s.T().Logf("SIMD_TAG=%s", tag)
+	ctx := context.Background()
+	r := s.CreateChainsRelayerAndChannel(ctx, feeMiddlewareChannelOptions())
+	s.T().Run("start relayer", func(t *testing.T) {
+		s.StartRelayer(r)
+	})
 
-	image, ok := os.LookupEnv("SIMD_IMAGE")
-	s.Require().True(ok)
-	s.T().Logf("SIMD_IMAGE=%s", image)
+}
 
-	s.T().Logf("Placeholder test")
-	s.Require().True(true)
+// feeMiddlewareChannelOptions configures both of the chains to have fee middleware enabled.
+func feeMiddlewareChannelOptions() func(options *ibc.CreateChannelOptions) {
+	return func(opts *ibc.CreateChannelOptions) {
+		opts.Version = "{\"fee_version\":\"ics29-1\",\"app_version\":\"ics20-1\"}"
+		opts.DestPortName = "transfer"
+		opts.SourcePortName = "transfer"
```

Comment:
nit: rename to avoid `dest`/`source` naming. Maybe in a followup issue?

Reply:
This one is trickier as it is part of the ibctest library.

Reply:
Is there already an issue created? Would you like me to open one?

Reply:
Yea you are right. Do you think it is too much work to request changes for? 

In the past I've spent a fair amount of time debugging the difference between source/destination, that it seems worthwhile to me to consider changing naming conventions if a library is going to have increased usage. Happy to let the maintainers decide if they want to do the changes, but the "source" and "destination" terminology doesn't make much sense when packet flow is bidirectional

Reply:
looking at the ibctest code I see the following

```golang
func (commander) CreateChannel(pathName string, opts ibc.CreateChannelOptions, homeDir string) []string {
	return []string{
		"rly", "tx", "channel", pathName,
		"--src-port", opts.SourcePortName,
		"--dst-port", opts.DestPortName,
		"--order", opts.Order.String(),
		"--version", opts.Version,
		"--home", homeDir,
	}
}
```

It looks like the field names are mappings to the go relayer arguments for channel creation so in this case I think the name makes sense.

I definitely agree that overall we shouldn't use `src/dst` for our chain names.